kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks.sh

```diff
@@ -0,0 +1,540 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# Description
+#  This sample script creates a WebLogic domain home on the Azure Kubernetes Service (AKS). 
+#  It creates a new Azure resource group, with a new Azure Storage Account and Azure File Share to allow WebLogic 
+#  to persist its configuration and data separately from the Kubernetes pods that run WebLogic workloads.
+#  Besides, it also generates the domain resource yaml files, which can be used to restart the Kubernetes 
+#  artifacts of the corresponding domain.
+#
+#  The Azure resource customized by editing create-domain-on-aks-inputs.yaml
+#  If you also want to customized WebLogic domain configuration, please edit kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-inputs.yaml. 
+#  or create a new copy and edit it, specify the file using "-d <your-domain-inputs.yaml>".
+#
+#  The following pre-requisites must be handled prior to running this script:
+#    * Environment has set up, with git, azure cli, kubectl and helm installed.
+#    * The doker hub account must have created, and checkouted Oracle WebLogic Server, we use 12.2.1.3 by default.
```

Comment:
The image should come from Oracle Container Registry and not Docker Hub.  Typo checkouted -> checked out